### ISSUE TITLE
Why the train subset and val subset are exactly the same set folder?

### ISSUE THREAD

**Issue by HulkMaker** [microsoft/Deep3DFaceReconstruction#138]
Why the train subset and val subset are exactly the same set folder?
Will it decrease the reconstruction performance?

**Comment by YuDeng** [comment]
The training set and validation set shown in the code are just examples. In your experiment, you have to set them to your custom folders. In that case the training set and validation set should be different folders.

**Comment by HulkMaker** [comment]
thanks for answer！
I set train and val subset different, train:val = 5:1, after 200k iterations, the landmark loss is still high as below, what's the issue?
![image](https://user-images.githubusercontent.com/5285649/111496533-5a430180-877b-11eb-81ef-a918017895c0.png)

**Comment by YuDeng** [comment]
Here the printed landmark loss are in pixel scale so that a value of 2 means 2-pixel distance to the ground truth, which is reasonable.

**Comment by HulkMaker** [comment]
great answer！
